plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,790 @@
+      - It can contain variables which will be replaced by their values.
+      - Currently supported are {{cluster}}, {{guestname}}, {{node}}, and {{vmid}},but more might be added in the future.
+      - Needs to be a single line, newline and backslash need to be escaped as '\n' and '\\' respectively.
+    type: str
+  performance:
+    description:
+      - Other performance-related settings.
+    type: str
+  pigz:
+    description:
+      - Use pigz instead of gzip when N>0.
+      - N=1 uses half of cores, N>1 uses N as thread count.
+    type: int
+    default: 0
+  pool:
+    description:
+      - Backup all known guest systems included in the specified pool.
+      - Can not be use with vmid and pool in same job
+    type: str
+  protected:
+    description:
+      - If true, mark backup(s) as protected.
+    type: bool
+  prune_backups:
+    description:
+      - Use these retention options instead of those from the storage configuration.
+    type: str
+    default: keep-all=1
+  quiet:
+    description:
+      - Be quiet.
+    type: bool
+    default: 0
+  remove:
+    description:
+      - Prune older backups according to 'prune-backups'.
+    type: bool
+    default: 1
+  repeat_missed:
+    description:
+      - If true, the job will be run as soon as possible if it was missed while the scheduler was not running.
+    type: bool
+    default: 0
+  schedule:
+    description:
+      - Backup schedule. The format is a subset of `systemd` calendar events.
+    type: str
+  script:
+    description:
+      - Use specified hook script.
+    type: str
+  starttime:
+    description:
+      - Job Start time.
+    type: str
+  stdexcludes:
+    description:
+      - Exclude temporary files and logs.
+    type: bool
+    default: 1
+  stop:
+    description:
+      - Stop running backup jobs on this host.
+    type: bool
+    default: 0
+  stopwait:
+    description:
+      - Maximal time to wait until a guest system is stopped (minutes).
+    type: int
+    default: 10
+  storage:
+    description:
+      - Store resulting file to this storage.
+    type: str
+  tmpdir:
+    description:
+      - Store temporary files to specified directory.
+    type: str
+  vmid:
+    description:
+      - The ID of the guest system you want to backup.
+      - Can not be use with vmid and pool in same job
+    type: str
+  zstd:
+    description:
+      - Zstd threads. N=0 uses half of the available cores,
+      - if N is set to a value bigger than 0, N is used as thread count.
+    type: int
+    default: 1
+
+extends_documentation_fragment:
+  - community.general.proxmox.actiongroup_proxmox
+  - community.general.proxmox.documentation
+  - community.general.attributes
+'''
+
+EXAMPLES = '''
+- name: List all backup jobs
+  community.general.proxmox_backup:
+  api_host: "node1"
+  api_user: user@realm
+  api_password: password
+  validate_certs: false
+  state: list
+  register: backup_result
```

Comment:
This example and all the examples below are indented wrongly. You must indent the module parameters by two spaces below the module name.

Reply:
Fix in 923fbd76014272b458eba79cdc257d9910718e23